server/parsec/components/memory/datamodel.py

```diff
@@ -52,6 +52,7 @@
     VerifyKey,
     VlobID,
 )
+from parsec._parsec_pyi.pki import PkiSignatureAlgorithm
```

Comment:
Be careful when using auto-complete for imports: it doesn't know a bout `parsec._parsec` and hence do the import from the `.pyi` typing stub files which is going to fail at runtime
```suggestion
from parsec._parsec import PkiSignatureAlgorithm
```